src/scrabble.js

```diff
@@ -1,6 +1,126 @@
+const scoreBoard = {
+  A: 1,
+  B: 3,
+  C: 3,
+  D: 2,
+  E: 1,
+  F: 4,
+  G: 2,
+  H: 4,
+  I: 1,
+  J: 8,
+  K: 5,
+  L: 1,
+  M: 3,
+  N: 1,
+  O: 1,
+  P: 3,
+  Q: 10,
+  R: 1,
+  S: 1,
+  T: 1,
+  U: 1,
+  V: 4,
+  W: 4,
+  X: 8,
+  Y: 4,
+  Z: 10
+}
+
 class Scrabble {
-  // Write your implementation here
+  constructor (word) {
+    this.word = word
+    this.multiplier = 1
+  }
+
+  // Scores for each letter
+  // static scoreBoard = { A: 1, B: 3, C: 3,
+  //                       D: 2, E: 1, F: 4,
+  //                       G: 2, H: 4, I: 1,
+  //                       J: 8, K: 5, L: 1,
+  //                       M: 3, N: 1, O: 1,
+  //                       P: 3, Q: 10, R: 1,
+  //                       S: 1, T: 1, U: 1,
+  //                       V: 4, W: 4, X: 8,
+  //                       Y: 4, Z: 10 }
+
+  /**
+   * Iterates over every character in a given word and sums the total score.
+   * @returns A score for the given word.
+   */
+  score () {
+    if (!this.isValid(this.word)) return 0
```

Comment:
i like the use of this if statement with the isValid function here - makes it easier to understand whats going on 👍

Reply:
I got the idea to write this extra method from Gui, it helped in breaking down the score method a bit more and improving readability.